### PR TITLE
Tests: port bz787736-pwconv-grpconv-skips-2nd-of-consecutive-failures

### PR DESCRIPTION
**Pull request by aborah-sudo**
https://pkgs.devel.redhat.com/cgit/tests/shadow-utils/tree/Regression/bz787736-pwconv-grpconv-skips-2nd-of-consecutive-failures

### CODE REVIEW

File: multihost_test/Bugzillas/test_bz_automation.py [hunk 1]
```diff
@@ -120,3 +120,25 @@ def test_bz672510(self, multihost):
...
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=787736
+        :steps:
+          1. If /etc/shadow (or /etc/gshadow) contains consecutive
+           bad lines pwconv only fixes the first, skipping the 2nd.
+        :expectedresults:
+          1. Should not succeed
+        """
+        execute_cmd(multihost, "echo 'example1:!!:15372:0:99999:7:::' >> /etc/shadow")
+        execute_cmd(multihost, "echo 'example2:!!:15372:0:99999:7:::' >> /etc/shadow")
+        # pwck first try should end with 2
```
Comment (ikerexxe) [inline multihost_test/Bugzillas/test_bz_automation.py:137]: Can you add in the comment that `2` means `error in one or more bad password entries`?

File: multihost_test/Bugzillas/test_bz_automation.py [hunk 2]
```diff
@@ -120,3 +120,25 @@ def test_bz672510(self, multihost):
...
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "pwck -r > /tmp/anuj")
```
Comment (ikerexxe) [inline multihost_test/Bugzillas/test_bz_automation.py:139]: Where are you checking that `pwck` returns 2?

Comment (aborah-sudo) [reply multihost_test/Bugzillas/test_bz_automation.py:139]: Here 2 means failure  :                  with pytest.raises(subprocess.CalledProcessError):

File: multihost_test/Bugzillas/test_bz_automation.py (hunk 1 above)
Comment (aborah-sudo) [reply multihost_test/Bugzillas/test_bz_automation.py:137]: Done

File: multihost_test/Bugzillas/test_bz_automation.py (hunk 2 above)
Comment (ikerexxe) [reply multihost_test/Bugzillas/test_bz_automation.py:140]: Thanks for clarifying.

File: multihost_test/Bugzillas/test_bz_automation.py [hunk 3]
```diff
@@ -120,3 +120,26 @@ def test_bz672510(self, multihost):
...
+        execute_cmd(multihost, "echo 'example1:!!:15372:0:99999:7:::' >> /etc/shadow")
+        execute_cmd(multihost, "echo 'example2:!!:15372:0:99999:7:::' >> /etc/shadow")
+        # pwck first try should end with 2
+        # 2 means error in one or more bad password entries
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "pwck -r > /tmp/anuj")
+        assert "no matching password file entry in /etc/passwd" in \
+               execute_cmd(multihost, "cat /tmp/anuj").stdout_text
+        execute_cmd(multihost, "pwconv")
+        # pwck second try should end with 0 or 2 if there is still missing homedirs
```
Comment (ikerexxe) [inline multihost_test/Bugzillas/test_bz_automation.py:144]: I think you can remove this comment

Comment (aborah-sudo) [reply multihost_test/Bugzillas/test_bz_automation.py]: Done

**Review by ikerexxe — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**